website/content/docs/k8s/connect/terminating-gateways.mdx

```diff
@@ -138,16 +138,13 @@ $ curl --request PUT --header "X-Consul-Token: $CONSUL_HTTP_TOKEN" --data @exter
 true
 ```
 
-### Update terminating gateway ACL token if ACLs are enabled
+### Update terminating gateway ACL role if ACLs are enabled
 
-If ACLs are enabled, update the terminating gateway acl token to have `service: write` permissions on all of the services
+If ACLs are enabled, update the terminating gateway acl role to have `service: write` permissions on all of the services
 being represented by the gateway:
 
 - Create a new policy that includes these permissions
-- Update the existing token to include the new policy
-
-~> The CLI command should be run with the `-merge-policies`, `-merge-roles` and `-merge-service-identities` so
-nothing is removed from the terminating gateway token
```

Comment:
should we keep this note? I think `-merge-roles` flag still seems appropriate

Reply:
these are only options on updating tokens.

Reply:
https://www.consul.io/commands/acl/tokens/update#command-options has them

https://www.consul.io/commands/acl/role/update#command-options does not

Reply:
Oh got it! Thanks for clarifying!